ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -1094,6 +1121,61 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-sdn
+- as: aws-ipi-shared-vpc-phz-tp-p2-f14
+  cron: 56 23 8,23 * *
+  remote_api: true
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @network-ovnkubernetes and not @fips'
+      FEATURE_SET: TechPreviewNoUpgrade
+      TAG_VERSION: '@4.14'
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-aws-ipi-shared-vpc-phz
+- as: aws-ipi-shared-vpc-phz-tp-p2-f28-destructive
+  cron: 36 0 25 * *
+  remote_api: true
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @network-ovnkubernetes and not @fips
+        and @destructive'
+      FEATURE_SET: TechPreviewNoUpgrade
+      TAG_VERSION: '@4.14'
+    test:
+    - chain: openshift-e2e-test-qe-destructive
+    workflow: cucushift-installer-rehearse-aws-ipi-shared-vpc-phz
+- as: aws-ipi-shared-vpc-phz-sts-fips-tp-p1-f14
```

Comment:
Suggest to update "p1" to "p2", too.

Reply:
Setting all profiles for this feature to p2? I thought to keep one on p1.

Reply:
Updated

Reply:
> Setting all profiles for this feature to p2? 

Two reasons:
1. We only selected server 4.next configurations as P1, tough P1 has different meaning in our test plan, but before a overall update, let us keep consistent.
2. These jobs are still TP, even follow the definition in our test plan doc, it also should be p2.